### PR TITLE
Files to make suse packages

### PR DESCRIPTION
**Pull request by aboe76**
these are the files needed to make suse packages.

### CODE REVIEW

File: pkg/suse/salt-cloud.changes [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- Added Parallels Support (X-Mission cloud provider)
+- Added Digital Ocean Support (cloud provider)
+- Updated Configuration Format
+- Posible to make Provider Aliases
+- Extending Profiles
+- Extending Providers
+
+Skipped 0.8.7 version because of packaging issues with
+downloading bootstrap script during install.
+This is packaged withing 0.8.8
```
Comment (s0undt3ch) [inline pkg/suse/salt-cloud.changes:26]: @aboe76 "within"?

Comment (aboe76) [inline pkg/suse/salt-cloud.changes:26]: Oops.. Sorry english is not my native language
Op 8 mei 2013 22:17 schreef "Pedro Algarvio" notifications@github.com het
volgende:

Comment (s0undt3ch) [inline pkg/suse/salt-cloud.changes:26]: No worries

**Comment by s0undt3ch** [comment]
Thanks @aboe76!

**Pull request merged**